src/resources/timezones.js

```diff
@@ -0,0 +1,5 @@
+import createResourceDescription from './util/createResourceDescription'
+
+export const timezones = createResourceDescription('timezones', {
```

Comment:
Q: currently you allow POST & DELETE, shouldn't we only allow GET methods for this endpoint?